packages/angular/src/package.json

```diff
@@ -22,6 +22,6 @@
     "@carbon/icons": "10.20.0",
     "@carbon/telemetry": "0.0.0-alpha.6",
     "flatpickr": "4.6.1",
-    "@ai-apps/styles": "0.0.0"
+    "@ai-apps/styles": "^2.145.0-next.16"
```

Comment:
is there any point in locking this dep to a specific version?

Reply:
Nope, but this _should_ be managed by lerna automatically